harper-dictionary-parsing/src/span.rs

```diff
@@ -11,9 +11,7 @@ pub struct Span {
 
 impl Span {
     pub fn new(start: usize, end: usize) -> Self {
-        if start > end {
```

Comment:
Let's keep this the way it is in `master`. You might be able to cherry-pick it.

Reply:
I don't remember making this change, which makes me assume clippy told me to do it. Regardless, I believe they have pretty much identical behavior, but the new one is better style imo.

Why do you want it the original way?

Reply:
Use the `panic!` method. It makes it _so_ much easier to debug related backtraces.